Assets/Scripts/UI/Overlay/OverlayMap.cs

```diff
@@ -351,11 +353,17 @@ private void Update()
             elapsed = 0f;
         }
 
+        if(currentOverlay != "None" && currentLayer != WorldController.Instance.cameraController.CurrentLayer)
+        {
+            Bake();
+            currentLayer = WorldController.Instance.cameraController.CurrentLayer;
+            elapsed = 0f;
+        }
         // TODO: Prettify.
```

Comment:
Stylecop `SA1515 : CSharp.Layout : A single-line comment must be preceded by a blank line or another single-line comment, or must be the first item in its scope. To ignore this error when commenting out a line of code, begin the comment with '////' rather than '//'. (Assembly-CSharp)`